.github/workflows/ci-uxd.yml

```diff
@@ -0,0 +1,143 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, v.** ]
+  pull_request:
+    branches: [ main, v.** ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+permissions:
+  contents: read
+  pull-requests: write
+  
+env:
+  solana_version: 1.8.11
```

Comment:
mango refers to solana version, ^1.8.1, but this should be fine, as long it doesnt break